resources/public/api/conf/2.0/schemas/historic_uk_non_fhl_period_summary_retrieve/response.json

```diff
@@ -0,0 +1,161 @@
+          "examples": "5000.99"
+        },
+        "otherIncome": {
+          "type": "number",
+          "description": "Total rents from property, ground rents and rent charges but not rent-a-room.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "taxDeducted": {
+          "type": "number",
+          "description": "Tax already deducted from the rental income.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "rentARoom": {
+          "type": "object",
+          "description": "Objects holding rents received for the period",
+          "properties": {
+            "rentsReceived": {
+              "type": "number",
+              "description": "Total rents received from properties.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+              "examples": "5000.99"
+            }
+          }
+        }
+      }
+    },
+    "expenses": {
+      "type": "object",
+      "description": "Object holding expenses for the period",
+      "properties": {
+        "premisesRunningCosts": {
+          "type": "number",
+          "description": "Rent, rates, insurance, ground rents and other costs.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "repairsAndMaintenance": {
+          "type": "number",
+          "description": "Property repairs and maintenance.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "financialCosts": {
+          "type": "number",
+          "description": "Loan interest and other financial costs.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "professionalFees": {
+          "type": "number",
+          "description": "Legal, management and other professional fees.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "costOfServices": {
+          "type": "number",
+          "description": "Cost of services provided, including wages.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "other": {
+          "type": "number",
+          "description": "Other allowable property expenses.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "consolidatedExpenses": {
+          "type": "number",
+          "description": "The sum of all expenses for the specified period.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
```

Comment:
Might it be worth adding a sentence here to make it explicit that you can't have consolidatedExpenses appear at the same time as all the other expenses? I know we already have an example for this scenario but it might be helpful to make this even clearer. What do you think?